docker/rholang-web/Dockerfile

```diff
@@ -34,6 +37,9 @@ RUN python3 manage.py migrate
 
 COPY docker/rholang-web/run-server .
 
+RUN python3 -c \
+    "from urllib.request import urlretrieve as wget; wget('https://github.com/django/django/raw/master/LICENSE', filename='/opt/rholangweb/rholang/main/static/django-license')"
```

Comment:
As in #239:
```
ADD https://github.com/django/django/raw/master/LICENSE /opt/rholangweb/rholang/main/static/django-license
```

Reply:
oops! that wasn't suppose to be in this PR at all.